### PR TITLE
Feature/users by group

### PR DESCRIPTION
**Pull request by sauloperez**
We need to return the users of a group for the frontend to render who's in it.

### CODE REVIEW

File: engines/account/Gemfile [hunk 1]
```diff
@@ -13,3 +13,4 @@ gemspec
 # To use debugger
 # gem 'debugger'
 gem 'shared', path: '../../engines/shared'
+gem 'basic_resources', path: '../../engines/basic_resources'
```
Comment (andresgutgon) [inline engines/account/Gemfile]: Ya no necesitas esto...dice @enricostano

File: engines/products/tags [hunk 2]
```diff
@@ -0,0 +1,86 @@
+!_TAG_FILE_FORMAT	2	/extended format; --format=1 will not append ;" to lines/
```
Comment (andresgutgon) [inline engines/products/tags]: git ignore `**/ctags` etc etc

Comment (sauloperez) [inline engines/products/tags]: Ups si si, me olvidé de quitarlo

File: engines/basic_resources/app/serializers/basic_resources/user_serializer.rb [hunk 3]
```diff
@@ -0,0 +1,10 @@
+module BasicResources
+  class UserSerializer < Shared::BaseSerializer
+    schema do
+      type 'user'
+
+      map_properties :id, :email, :first_name, :last_name, :username,
+        :password_digest, :created_at, :updated_at
```
Comment (enricostano) [inline engines/basic_resources/app/serializers/basic_resources/user_serializer.rb]: `:password_digest` :red_circle:

**Pull request merged**